lib/express-handlebars.js

```diff
@@ -176,11 +176,9 @@ ExpressHandlebars.prototype.render = function (filePath, context, options) {
     ]).then(function (templates) {
         var template = templates[0],
             partials = templates[1],
+            helpers  = options.helpers || this.helpers,
             data     = options.data;
 
-        var helpers = options.helpers ||
-                utils.extend({}, this.handlebars.helpers, this.helpers);
```

Comment:
I removed the merge of `this.handlebars.helpers` since Handlebars will do this merge automatically at render time.